examples/prometheus/README.md

```diff
@@ -0,0 +1,137 @@
+# Monitoring STUNner with Prometheus and Grafana
+
+This tutorial demonstrates monitoring of a STUNner instance with [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/).
+
+In this demo you will learn how to:
+* monitor STUNner with [Prometheus](https://prometheus.io/)
+* how to visualize STUNner metrics using [Grafana](https://grafana.com/)
+
+## Installation
+
+### Prerequisites
+
+* The tutorial assumes a fresh STUNner installation in a namespace called `stunner`; see the [STUNner installation and configuration guide](/doc/INSTALL.md).
+* You need a web browser to run this tutorial.
+
+### Enable STUNner metrics endpoint
+
+After installing STUNner, enable the STUNner metrics endpoint:
+```console
+kubectl -n stunner patch gatewayconfigs.stunner.l7mp.io stunner-gatewayconfig --patch '{"spec": {"metricsEndpoint": "http://0.0.0.0:8080/metrics" }}' --type=merge
+```
+
+Patch STUNner deployment to enable the monitoring port (required by the pod monitor):
+```console
+kubectl -n stunner patch deployment stunner --patch '{"spec":{"template":{"spec":{"containers":[{"name":"stunnerd","ports":[{"name": "web","containerPort": 8080 }]}]}}}}'
+```
+
+### Deploy Prometheus and Grafana
+
+Deploy Prometheus with `helm`:
+```console
+helm install prometheus stunner/stunner-prometheus
+```
+
+The helm chart creates the namespace `monitoring` and installs Prometheus along with the prometheus-operator, and Grafana.
+
+## Usage
+
+### Observe metrics on the Prometheus dashboard (optional)
+By default Prometheus dashboard is not accessable outside of the cluster. No worries if that is the case -- you can still observe the metrics with Grafana.
+
+In certain deployments (e.g., local minikube) you can open the Prometheus dashboard by navigating your browser to prometheus service IP and port `9090`.  For example, if the prometheus service cluster-IP is `10.103.67.2`, navigate to `http://10.103.67.2:9090`.
+
+To get the service IP, use this command: `kubectl get svc -n monitoring prometheus -o custom-columns=:.spec.clusterIP --no-headers`; to have a clickable link: `echo -n "http://$(kubectl get svc -n monitoring prometheus -o custom-columns=:.spec.clusterIP --no-headers):9090"`
+
+If everything goes well, the Prometheus dashboard opens:
+TODO add img
+
+Next, we observe the `stunner_allocations_active` metrics:
```

Comment:
What's this? Is it related to what we measure or to how we visualize it in grafana (e.g. dashboard)?